src/test/unit/controls/Gantt/Track-spec.js

```diff
@@ -190,6 +190,32 @@ describe("Gantt -> Track", () => {
                 }
             );
         });
+        it("Executes callback when, clicked on truncated label and onClick function is not provided", (done) => {
```

Comment:
nit: Can we write like below for all event handler test cases

**E.g**..

```js
describe("when small label and onLabelClick function is provided", (done) => {
      it("should disable onLabelClick functionality", (done) =>
         ......................
      } 
}

describe("when small label is provided and onLabelClick function is not provided", (done) => {
      it("should disable onLabelClick functionality", (done) =>
         ......................
      } 
}

describe("when large label and onLabelClick function is provided", (done) => {
      it("should enable onLabelClick functionality", (done) =>
         ......................
      } 
}

describe("when large label is provided and onLabelClick function is not provided", (done) => {
      it("should enable onLabelClick functionality", (done) =>
         ......................
      } 
}
```

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/242/commits/3d75d6b1109fb3da7bad04f51b1228bd6bcaaf93#diff-7fb4a16b40433875e7e73811189b4f84R164)